### PR TITLE
home-page

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
Home-Page

### CODE REVIEW

File: screens/Home-Page/home.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        
+        </div>
+       
+   
+
+
+    </div>    
+
+        
+
```
Comment (anshulraheja) [inline screens/Home-Page/home.html]: Remove unnecessary spaces

File: screens/Home-Page/home.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <link rel="stylesheet" href="https://myui-sumit.netlify.app/style.css">
+	
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+</head>
+<body>
+    <div class="containr">
+		<nav>
+			<div class="logo">Fashion-Mart</div>
+			<input class="search-data" type="text" placeholder="serach">
+			<i class="fal fa-search"></i>
```
Comment (anshulraheja) [inline screens/Home-Page/home.html:18]: This icon is not visible on screen

File: screens/Home-Page/home.css [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    
+      font-weight: 500;
+    }
+
+    
+    span {
+      margin: 10px;
+    }
+    input {
+      margin-left: 2rem;
```
Comment (anshulraheja) [inline screens/Home-Page/home.css:132]: use classes or child selectors access elements

**Review by anshulraheja — APPROVED** [review]

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 👍

File: screens/Home-Page/home.html (hunk 1 above)
Comment (SumitSaundadkar) [reply screens/Home-Page/home.html]: thank you anshul.

File: screens/Home-Page/home.css (hunk 3 above)
Comment (SumitSaundadkar) [reply screens/Home-Page/home.css:132]: done.

File: screens/Home-Page/home.html (hunk 2 above)
Comment (SumitSaundadkar) [reply screens/Home-Page/home.html:18]: done.

**Pull request merged**